src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -249,13 +250,16 @@ public String getQuestionResultStatisticsCsv(
             if (shouldSelfBeExcludedFromRankings(question)) {
                 fragments.append(',').append(userAverageExcludingSelfText);
             }
+            fragments.append(',');
+            fragments.append(StringHelper.join(",", ranks));
             fragments.append(Const.EOL);
         }
         String rankQuestionHeaderSelf = shouldSelfBeExcludedFromRankings(question)
                 ? ", Average Rank Excluding Self"
                 : "";
 
-        return "Team, Recipient, Average Rank" + rankQuestionHeaderSelf + Const.EOL + fragments + Const.EOL;
+        return "Team, Recipient, Average Rank" + rankQuestionHeaderSelf + ", Ranks Received" + Const.EOL
```

Comment:
`Ranks received` should be before `Average rank` to keep CSV results consistent with the HTML results.